vb-be/src/modules/auth/auth.service.ts

```diff
@@ -85,12 +84,26 @@ export class AuthService {
 
   async refreshTokens(
     accessToken: string,
-    userInfo: UserType,
+    user: ITokenUser,
   ): Promise<string[]> {
-    if (!accessToken || !userInfo['email'] || !userInfo['id']) {
-      throw new ErrorHandler(HttpStatus.Error, ErrorText.InvalidRTArg);
-    }
     await this.delAsyncJwtClient(accessToken);
-    return await this.createTokens(userInfo);
+    return await this.createTokens(user);
+  }
+
+  async verifyToken(token: string, type: 'access' | 'refresh'): Promise<any> {
```

Comment:
use Enums here. keep it in the `auth.constants` like,
```
export enum TokenTypes {
Access = 'access',
Refresh = 'refresh,
}
```
use it like,
(token: string, type: TokenType)